theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -34,6 +34,49 @@ def find_changed_packages(diff_range) {
     return changed_packages.split()
 }
 
+def find_changed_debs(diff_range) {
+    def changed_debs = []
+
+    def changed_dependencies = find_added_or_changed_files(diff_range, 'dependencies/*/*/changelog').split()
+    def changed_plugins = find_added_or_changed_files(diff_range, 'plugins/*/changelog').split()
+    def changed_core = find_added_or_changed_files(diff_range, 'debian/*/*/changelog').split()
+
+    for(core in changed_core) {
+        core_split = core.split('/')
```

Comment:
I think you can use something link this:

```suggestion
        (folder, os, project) = core.split('/')
```

and then use the better names below.

```console
groovy:000> s = "debian/focal/foreman/changelog"
===> debian/focal/foreman/changelog
groovy:000> (type, os, project) = s.split('/')
===> [Ljava.lang.String;@dc7df28
groovy:000> type
===> debian
groovy:000> project
===> foreman
groovy:000> os
===> focal
```